framework/contracts/native/registry/src/contract.rs

```diff
@@ -46,11 +46,9 @@ pub fn instantiate(deps: DepsMut, _env: Env, _info: MessageInfo, msg: Instantiat
     cw_ownable::initialize_owner(deps.storage, deps.api, Some(&admin))?;
 
     // Save the abstract namespace to the Abstract admin account
-    NAMESPACES.save(
-        deps.storage,
-        &Namespace::new(ABSTRACT_NAMESPACE)?,
-        &ABSTRACT_ACCOUNT_ID,
-    )?;
+    let namespace = Namespace::new(ABSTRACT_NAMESPACE)?;
+    NAMESPACES.save(deps.storage, &namespace, &ABSTRACT_ACCOUNT_ID)?;
+    REV_NAMESPACES.save(deps.storage, &ABSTRACT_ACCOUNT_ID, &namespace)?;
```

Comment:
Not related to this PR, but had to fix it to fetch abstract account by namespace. We didn't save abstract account in REV_NAMESPACE